issueshark/backends/github.py

```diff
@@ -148,47 +148,38 @@ def _process_events(self, system_id, mongo_issue):
         :param mongo_issue: object of our issue model
         """
         # Get all events to the corresponding issue
-        target_url = '%s/%s/events' % (self.config.tracking_url, system_id)
+        target_url = '%s/%s/events' % (self.config.tracking_url, self.issue_id)
         events = self._send_request(target_url)
 
         # Go through all events and create mongo objects from it
         events_to_store = []
         for raw_event in events:
-            created_at = dateutil.parser.parse(raw_event['created_at'])
+            created_at = dateutil.parser.parse(raw_event['created_at']).replace(tzinfo=None)
```

Comment:
maybe a short function for this could be in pycoshark so that all plugins can use that.